aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -615,9 +674,14 @@ module AptosFramework::Stake {
         pool_address: address,
         amount: u64,
         owner_cap: &OwnerCapability,
-    ) acquires StakePool, StakePoolEvents, ValidatorSet {
+    ) acquires StakePool, StakePoolEvents, ValidatorSet, ValidatorSetConfiguration, ValidatorSetVotingPower {
         assert!(owner_cap.pool_address == pool_address, Errors::invalid_argument(ENOT_OWNER));
 
+        let validator_set_voting_power = borrow_global_mut<ValidatorSetVotingPower>(@CoreResources);
+        validator_set_voting_power.withdrawing = validator_set_voting_power.withdrawing + (amount as u128);
+        let validator_set_config = borrow_global<ValidatorSetConfiguration>(@CoreResources);
+        validate_voting_power_withdrawal(validator_set_voting_power, validator_set_config);
```

Comment:
here too, we should not count pools not in the validator set

Reply:
Updated the logic according to the suggestion and added unit tests to make sure we don't count inactive validator's stake in ValidatorSetVotingPower#leaving.